opencodelists/tests/fixtures.py

```diff
@@ -347,6 +347,7 @@ def build_fixtures():
         owner=organisation,
         name="Old-style Codelist",
         coding_system_id="snomedct",
+        coding_system_database_alias=_get_coding_system("snomedct").database_alias,
```

Comment:
We're calling `_get_coding_system("snomedct").database_alias` a lot!  Can we extract it into a variable?